### ISSUE TITLE
cat is not a recognized entity type

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#2074]
### Description
**cat** does not seem to be recognized in 1.14.x; it is recognized in 1.13.2.

### Steps to Reproduce
Start server.

### Expected Behavior
No error.

### Errors 
```
[12:29:23] [Server thread/ERROR]: [Skript] Can't compare an entity with 'a cat' (save-my-pets.sk, line 11: if shooter's target is a cat:')
[12:29:23] [Server thread/ERROR]: [Skript] can't understand this event: 'on damage of a cat' (save-my-pets.sk, line 26: on damage of a cat:')
```
### Server Information
```
>version
[12:41:41] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-4f661b2-585b2eb (MC: 1.14.1) (Implementing API version 1.14.1-R0.1-SNAPSHOT)
[12:41:41] [Server thread/INFO]: You are 1 version(s) behind
>version Skript
[12:41:50] [Server thread/INFO]: Skript version 2.4-alpha1
[12:41:50] [Server thread/INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[12:41:50] [Server thread/INFO]: Website: https://skriptlang.github.io/Skript
[12:41:50] [Server thread/INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, Snow-Pyon, Pikachu920, Nicofisi, tim740, Tuke-Nuke, TheLimeGlass, xXAndrew28Xx, Sashie, RoyCurtis, nfell2009, Syst3ms and Blueyescat
```
### Skript
```
on shoot:
        if shooter's target is a cat:
                cancel event
                send "Don't shoot my cat!" to the shooter
                strike lightning effect at the shooter
                strike lightning effect at the shooter
                strike lightning effect at the shooter
                create a fake explosion at the shooter
        if shooter's target is a dog:
                cancel event
                send "Don't shoot my dog!" to the shooter
                strike lightning effect at the shooter
                strike lightning effect at the shooter
                strike lightning effect at the shooter
                create a fake explosion at the shooter

on damage of a cat:
        cancel event
        send "Don't touch my cat!" to the attacker
        strike lightning effect at the attacker
        strike lightning effect at the attacker
        strike lightning effect at the attacker
        create a fake explosion at the attacker

on damage of a dog:
        cancel event
        send "Don't touch my dog!" to the attacker
        strike lightning effect at the attacker
        strike lightning effect at the attacker
        strike lightning effect at the attacker
        create a fake explosion at the attacker
```

**Comment by TheBentoBox** [comment]
Already reported as part of #2064.
